File: 250222/절댓값으로 변경/find-the-absolute-value.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;
        
        st = new StringTokenizer(br.readLine());

        int n = Integer.parseInt(st.nextToken());
        int arr[] = new int[n];

        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < n; i++) {
            arr[i] = Math.abs(Integer.parseInt(st.nextToken()));
            sb.append(arr[i]).append(" ");
        }
        System.out.println(sb);
    }
}

File: 250222/절댓값으로 변경/find-the-absolute-value.cpp
#include <iostream>
#include <algorithm>

using namespace std;


int main() {
    int n;
    int arr[50];
    cin >> n;

    for (int i = 0; i < n; i++) {
        int temp;
        cin >> temp;
        arr[i] = abs(temp);
    }
    for (int i = 0; i < n; i++)
        cout << arr[i] << " ";
    return 0;
}